airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml

```diff
@@ -44,6 +44,9 @@ data:
               - "ads_insights_region"
               - "ads_insights_dma"
               - "ads_insights_action_product_id"
+      3.0.0:
+        message: "All Custom Insights Streams that uses body_asset, call_to_action_asset, description_asset, image_asset, link_url_asset, title_asset or video_asset as breakdown now have updated schemas. Users will need to retest source configuration, refresh the source schema and reset affected streams after upgrading. "
```

Comment:
It's a shame that we can't target streams more specifically here as everyone will have to opt-in instead of just those with this specific situation. 

@erohmensing do you know if there is a way to scope that better? Basically, the user can enter `custom_insights.X.name` which will create a stream named `custom<custom_insights.X.name>`. If we can scope even more/in another way, targeting some fields from the json schema would also be good.

Reply:
if I user does not use body_asset, call_to_action_asset, description_asset, image_asset, link_url_asset, title_asset or video_asset they dont need to reset?

Reply:
@katmarkham, you don't need a reset as your stream schema doesn't have fields above. The source returns stream schema according to provided breakdowns in the custom stream config.

Reply:
Maybe a broader discussion for my personal learning: do we need to make it a breaking change if we have these options now? 

![image](https://github.com/airbytehq/airbyte/assets/3360483/62934889-2d1a-499a-9c87-b0a784312379)

If so, I would keep the opt-in mechanism only for when the config is changing. In our case, it is only a schema change

Reply:
As @darynaishchenko mentioned in another comment, this would still be a breaking change because of the PK changing

Reply:
> @erohmensing do you know if there is a way to scope that better? Basically, the user can enter custom_insights.X.name which will create a stream named custom<custom_insights.X.name>. If we can scope even more/in another way, targeting some fields from the json schema would also be good.

I know this is over, but wanted to add - unfortunately I looked into this when we were creating it, but making some sort of regex matching or globbing proved to be something we couldn't implement because we are very lenient with characters in the stream names, meaning any character like * could be a wildcard or an actual character in the stream name.